src/yanfeishao/cs555/constant/ErrorInfo.java

```diff
@@ -39,7 +39,7 @@
     /**
      * The constant US10_FAMILY.
      */
-    public static final String US10_FAMILY = "Marriage should be at least 14 years after birth of both spouses";
+    public static final String US10_FAMILY = "Marriage should be at least 14 years after birth of both spouses, family [%s] wife age [%s] and husband age [%s] at marriage";
     /**
```

Comment:
This consant is no longer used it has been replaced with US10_Person 
that has the requested additional information.

On 10/18/2015 10:44 PM, JackieDreamy wrote:
> In src/yanfeishao/cs555/constant/ErrorInfo.java [1]:
> 
>> @@ -39,7 +39,7 @@
>> /**
>> * The constant US10_FAMILY.
>> */
>> - public static final String US10_FAMILY = "Marriage should be at
> least 14 years after birth of both spouses";
>> + public static final String US10_FAMILY = "Marriage should be at
> least 14 years after birth of both spouses, family [%s] wife age [%s]
> and husband age [%s] at marriage";
>> /**
> 
> Please add more information for one person, like wife [Personal Id %s
> Name: %s]
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/JackieDreamy/AgileSprint/pull/49#discussion_r42332954
> [2] https://github.com/JackieDreamy/AgileSprint/pull/49/files#r42332954